addons/laser/XEH_postInit.sqf

```diff
@@ -47,6 +47,71 @@ if (hasInterface) then {
+["ace_laserOn", {
+    params ["_uuid", "_args"];
+    private _vehicle = _args select 0;
+    if (hasPilotCamera _vehicle) then {
+        [_vehicle] call FUNC(laserPointTrack);
+    };
+}] call CBA_fnc_addEventHandler;
+
+["AllVehicles", "init", {
+    params ["_vehicle"];
+    private _hasPilotCamera = hasPilotCamera _vehicle;
+    private _hasLaserSensor = isClass (configfile >> "CfgVehicles" >> (typeOf _vehicle) >> "Components" >> "SensorsManagerComponent" >> "Components" >> "LaserSensorComponent");
+
+
+    if ( (_hasPilotCamera) && (_hasLaserSensor) ) then {
+        _vehicle setVariable [QGVAR(hasLaserSpotTracker), true];
+        _vehicle setVariable [QGVAR(laserSpotTrackerOn), false];
+        private _actionOff = ["LSTOff", localize LSTRING(LSTOff), "", {[_this select 0] call FUNC(toggleLST)}, {(_this select 0) getVariable [QGVAR(laserSpotTrackerOn), false]}] call ace_interact_menu_fnc_createAction;
+        [_vehicle, 1, ["ACE_SelfActions"], _actionOff] call ace_interact_menu_fnc_addActionToObject;
+        private _actionOn = ["LSTOn", localize LSTRING(LSTOn), "", {[_this select 0] call FUNC(toggleLST)}, {!((_this select 0) getVariable [QGVAR(laserSpotTrackerOn), false])}] call ace_interact_menu_fnc_createAction;
+        [_vehicle, 1, ["ACE_SelfActions"], _actionOn] call ace_interact_menu_fnc_addActionToObject;
+    };
+
+    private _foundLaser = 0;
+    private _weapons = weapons _vehicle;
+    {
+        _weapons = _weapons + (_vehicle weaponsTurret _x);
+    } forEach (allTurrets _vehicle);
+
+    if (_weapons find "Laserdesignator_mounted" > -1) then {
+        _foundLaser = 1;
+    };
+    if (_weapons find "Laserdesignator_pilotCamera" > -1) then {
+        _foundLaser = 2;
+    };
+
+    if (_foundlaser == 0) exitWith {};
+    _vehicle setVariable [QGVAR(hasMarkerLaser), true];
+    _vehicle setVariable [QGVAR(laserMarkerOn), false];
+    
+    private _actionOff = ["LaserMarkerOff", localize LSTRING(laserMarkOff), "", {[_this select 0] call FUNC(toggleMarker)}, {(_this select 0) getVariable [QGVAR(laserMarkerOn), false]}] call ace_interact_menu_fnc_createAction;
+    [_vehicle, 1, ["ACE_SelfActions"], _actionOff] call ace_interact_menu_fnc_addActionToObject;
+    private _actionOn = ["LaserMarkerOn", localize LSTRING(laserMarkOn), "", {[_this select 0] call FUNC(toggleMarker)}, {!((_this select 0) getVariable [QGVAR(laserMarkerOn), false])}] call ace_interact_menu_fnc_createAction;
+    [_vehicle, 1, ["ACE_SelfActions"], _actionOn] call ace_interact_menu_fnc_addActionToObject;
+    
+}, nil, nil, true] call CBA_fnc_addClassEventHandler;
+
+
+["CAManBase", "init", {
+    params ["_unit"];
+    findDisplay 46 displayAddEventHandler ["KeyDown", {
+        params ["_displayorcontrol", "_key", "_shift", "_ctrl", "_alt"];
+        if (_ctrl) then {_key = _key + 486539264};
+        if (_shift) then {_key = _key + 704643072};
+        if (_alt) then {_key = _key + 939524096};
+        private _lightKeys = actionKeys "headlights";
```

Comment:
This does not work. Floats too big. `actionKeys` command is unusable. This method cannot be saved. Rethink it from the beginning.